pkg/controller/migmigration/annotation.go

```diff
@@ -235,7 +235,14 @@ func (t *Task) annotatePods(client k8sclient.Client) (ServiceAccounts, error) {
 			if pod.Labels == nil {
 				pod.Labels = make(map[string]string)
 			}
+
+			labels := t.Owner.GetCorrelationLabels()
```

Comment:
:+1:  Great use of correlation labels instead of a new label.

Not sure why this code is singled out as a separate block (surrounded by blank lines).